wazo_dird_plugin_odoo/schemas.py

```diff
@@ -9,6 +9,8 @@
 
 class SourceSchema(BaseSourceSchema):
     server = fields.String(required=True)
+    model = fields.String(required=True)
+    username = fields.String(required=True)
```

Comment:
What is the difference between userid and username? Looks like we don't need it anymore?

Reply:
Yes I deleted userid, that is obsolete and replaced by username